### ISSUE TITLE
An element with aria-hidden=true contains focusable content

### ISSUE THREAD

**Issue by Horlix** [canonical-web-and-design/ubuntu.com#11905]
The hidden element and its contents are not voiced by a screen reader, but the contents contain an element with tabindex set or active a, button, input, select and textarea controls. The user can tab to these or focus via touch and hears only silence.

Full set of URLs provided

[An element with aria-hidden=true contains focusable content.csv](https://github.com/canonical-web-and-design/ubuntu.com/files/9213497/An.element.with.aria-hidden.true.contains.focusable.content.csv)

**Comment by immortalcodes** [comment]
Adding a bit of context here:
- The issue here is that the elements within a hidden container are not announced by screen readers but can still receive keyboard focus or touch interaction.
- Hidden containers may be created using CSS properties such as `display: none;` or `visibility: hidden;`, or ARIA properties like `aria-hidden="true"`. 
- Focusable elements within these containers, such as links (`<a>`), buttons (`<button>`), input fields (`<input>`), dropdowns (`<select>`), and text areas (`<textarea>`), can still be accessed through keyboard navigation or touch, but the screen reader does not announce their content, leading to silence and user confusion. 
- This issue can arise from improper management of visibility and focus. 

Possible solution would be:
- Better management of ARIA attributes, [tabindex](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) could be used to focus/not focus on certain elements.
- JS scripts should also handle tabindex while they are setting aria-hidden as false

**Comment by petesfrench** [comment]
I think we can close this issue and here is why:

I reviewed the attached document of accessibility infringements and all occurrences are within `/tutorial`. I examined some of the pages (which use a re-usable template) and the only places where `aria-hidden` is used is in the navigation. This would mean it effects all pages, not just `/tutorials`.

From this we can assume the issue that was previously isolated to `/tutorials` has been resolved.
